src/odoc/html_page.ml

```diff
@@ -16,6 +16,42 @@
 
 open StdLabels
 
+let write_html_to_channel : Pervasives.out_channel -> Tyxml.Html.doc -> unit =
+    fun channel page ->
+
+  let ns name = (Markup.Ns.html, name) in
```

Comment:
Is this deconstructing the tuple on the right? If so, what's the difference in the bound `name` here?

Reply:
"Deconstruction" or pattern-matching is always on the left side, _e.g._ see the [pattern position](https://caml.inria.fr/pub/docs/manual-ocaml/expr.html#let-binding) in let-bindings. In this case it's a tuple _construction_ with the `name` value as the second field.

Reply:
Yes, I understand that, but wouldn't this be the same as binding `Markup.Ns.html` to `ns` and letting `name` be? 🤔 

That's where my confusion arose.

Reply:
It's a function that expands `name` by adding the HTML namespace. If you suggest a clearer way to write it, I can change it.

Reply:
A-ha! I did not see that the left hand side is being evaluated as well.

This clears it up 👍🏼